py3status/modules/khal_calendar.py

```diff
@@ -7,6 +7,7 @@
     date_end: Until which datetime the module searches for events (default 'eod')
     format: display format for this module (default '{appointments}')
     output_format: khal conform format for displaying event output (default '{start-time} {title}')
+    limit: limit the number of returned calendar entries (default: no limit)
```

Comment:
Should be `(default None)`.

Reply:
I would argue that '(default None)', albeit technically correct, is not a helpful stanza of documentation, since it is ambiguous whether it means 'unlimited' or 'zero' in this context.

Reply:
Thing is, CI and code requirements of py3status require that the documented default is the same as the one defined below

Reply:
oh. In that case, maybe the parameter or the description can be rephrased to reduce the ambiguity instead.

maybe:
`max_results: an upper bound for the number of returned calendar entries (default: None)`